cloud/cloud.py

```diff
@@ -1,18 +1,29 @@
+import libcloud
+import toml
+
 import cloud
 from cloud import registry as reg
 from cloud import Instance
 
 
-def connect(arg):
-
-  if isinstance(arg, str):
+def connect(arg=None):
```

Comment:
imo, this should ONLY take path to config

Reply:
imho, it should NOT